app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/ScheduleComposables.kt

```diff
@@ -478,32 +476,15 @@ private fun ContextMenu(
     hasAlarm: Boolean,
     shouldShowShareSubMenu: Boolean,
     textColor: Color,
-    cardPosition: Offset,
     pressPosition: Offset,
     onMenuItemClick: (SessionInteractionType) -> Unit,
     onCollapseRequest: () -> Unit,
 ) {
     var showShareSubMenu by remember { mutableStateOf(false) }
 
-    // The UI stack above the schedule content includes:
-    // 1. System status bar (~24dp)
-    // 2. App toolbar (~56dp)
-    // 3. Progress bar (variable) or horizontal scrolling line
-    // 4. Room names header
-    val systemUiOffset = 132.dp // Combined height of all UI elements above schedule
-
-    // Times layout column width
-    val timeColumnWidth = dimensionResource(R.dimen.schedule_time_column_layout_width)
-
-    val density = LocalDensity.current
-    val relativeX = with(density) { pressPosition.x.toDp() }
-    val relativeY = with(density) { pressPosition.y.toDp() }
-    val cardPositionX = with(density) { cardPosition.x.toDp() }
-    val cardPositionY = with(density) { cardPosition.y.toDp() }
-
-    val menuXOffset = cardPositionX + relativeX - timeColumnWidth
-    val menuYOffset = cardPositionY + relativeY - systemUiOffset
-    val menuOffset = DpOffset(menuXOffset, menuYOffset)
+    val menuOffset = with(LocalDensity.current) {
+        DpOffset(pressPosition.x.toDp(), pressPosition.y.toDp())
+    }
```

Comment:
I tried to move this calculation of the `DpOffset` value into the call site (`SessionCard`) so only the final `menuOffset` can be passed into this composable.

Interestingly enough, the context menu position is wrong again in that case. Can you explain why the calculation must happen in the `ContextMenu` composable?

Reply:
Nice. I tried moving it into `pointerInteropFilter`.